integration/script/run-test

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -i
+
+function start_test_demo {
```

Comment:
Can we use node script instead of bash :D

Reply:
I prefer to do this later. 

Since the JS SDK using the bash, I can refer to that one. If changing to node script, I will need to spend more time to learn and update, since no reference I assume.

Reply:
Node script is easier to do and which we all know well. I don't think we should create any new bash script.

Reply:
Updated.